addons/print/wizard/print_order_sendnow_wizard.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+from openerp import api, fields, models
+
+
+class print_order_sendnow_wizard(models.TransientModel):
+
+    _name = 'print.order.sendnow.wizard'
+    _rec_name = 'create_date'
+
+    # --------------------------------------------------
+    # MODEL FIELDS
+    # --------------------------------------------------
+    print_order_sendnow_line_wizard_ids = fields.One2many('print.order.sendnow.line.wizard', 'print_order_sendnow_wizard_id', string='Lines')
+
+    # --------------------------------------------------
+    # METHODS
+    # --------------------------------------------------
+    @api.model
+    def default_get(self, fields):
+        """ create the lines on the wizard """
+        res = super(print_order_sendnow_wizard, self).default_get(fields)
+        active_ids = self._context.get('active_ids', [])
+        if active_ids:
+            # create order lines
+            lines = []
+            for rec in self.env['print.order'].browse(active_ids):
+                if rec.state != 'sent':
+                    lines.append((0, 0, {
+                        'res_model': rec.res_model,
+                        'ink' : rec.ink,
+                        'partner_id' : rec.partner_id.id,
+                        'provider_id' : rec.provider_id.id,
+                        'state' : rec.state,
+                        'order_id' : rec.id
+                    }))
+            res['print_order_sendnow_line_wizard_ids'] = lines
```

Comment:
Technically you should check that print_order_sendnow_line_wizard_ids is in fields before adding it to res.